src/logic/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -43,6 +45,12 @@ export default (safeAddress: string): ThunkAction<Promise<void>, AppReduxState,
     if (incomingTransactions.get(safeAddress).size) {
       dispatch(addIncomingTransactions(incomingTransactions))
     }
+
+    const moduleTransactions = await loadModuleTransactions(safeAddress)
+
+    if (moduleTransactions) {
```

Comment:
the default response for this is `[]`, I think you wanted to check .length

Reply:
fixed, thanks